Containers/talk/Dockerfile

```diff
@@ -45,30 +46,23 @@ RUN set -ex; \
     \
     touch \
         /etc/nats.conf \
-        /etc/signaling.conf \
-        /etc/turnserver.conf; \
+        /etc/signaling.conf; \
     echo "listen: 127.0.0.1:4222" | tee /etc/nats.conf; \
     mkdir -p \
-        /var/tmp \
-        /var/lib/turn \
         /var/log/supervisord \
         /var/run/supervisord; \
-    chown talk:talk -R \
+    chown eturnal:eturnal -R \
         /usr \
         /etc/janus \
+        /opt/eturnal \
         /etc/nats.conf \
         /etc/signaling.conf \
-        /etc/turnserver.conf \
-        /var/lib/turn \
         /var/log/supervisord \
         /var/run/supervisord;
 
-# Set default talk port https://github.com/nextcloud/all-in-one/issues/1011
-ENV TALK_PORT=3478
-
-USER talk
+USER eturnal
 ENTRYPOINT ["/start.sh"]
 CMD ["supervisord", "-c", "/supervisord.conf"]
 
-HEALTHCHECK CMD (nc -z localhost 8081 && nc -z localhost 8188 && nc -z localhost 4222 && nc -z localhost "$TALK_PORT" && nc -z "$NC_DOMAIN" "$TALK_PORT") || exit 1
+HEALTHCHECK CMD (nc -z localhost 8081 && nc -z localhost 8188 && nc -z localhost 4222 && nc -z localhost "$TALK_PORT" && nc -z "$NC_DOMAIN" "$TALK_PORT" && eturnalctl status) || exit 1
```

Comment:
Does eturnalctl stauts return exitcode 0 if running correctly and something else if not?

Reply:
Yes, it returns status with exit code 0. If not, it returns exit code 1
https://github.com/erlware/relx/blob/8c571fde08aaa2d174e69fb632ba98312f3765a0/priv/templates/extended_bin#L251-L256

Reply:
Okay cool! Thanks! :)